packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -192,16 +220,21 @@ class DatePicker extends React.Component {
   }
 
   handleOnInputFocus(event) {
-    if (this.onInputFocus) {
-      this.onInputFocus(event);
-    }
+    this.handleFocus(event);
 
     if (!this.isDefaultDateAcceptable) {
       this.handleChange(null, event);
       this.isDefaultDateAcceptable = true;
     }
   }
 
+  handleFocus(event) {
+    // Handle focus only if focus is gained from outside of the entire date picker component.
+    if (this.props.onFocus && !this.datePickerContainer.current.contains(event.relatedTarget)) {
```

Comment:
The comment above mentions IE10/11 issues with `relatedTarget`. Should we be doing a the same presence here as above?

Reply:
Ah, yes. I need to do something similar for IE as above but just not the same as above. I can't use activeElement since it references the current active element. I need to get the _previous_ active element. I haven't found a native way to do so I may have to keep track of it internal.

Reply:
Added change to make it work in IE. https://github.com/cerner/terra-framework/pull/588/commits/dc4b6b92c993c0c6ea2aeec03e1844783473ba99 and https://github.com/cerner/terra-framework/pull/588/commits/4253be45c6ac86ee205bff37876d6ffda377e5fd